opendebates/templates/opendebates/snippets/idea.html

```diff
@@ -1,6 +1,8 @@
 {% load i18n %}
 {% load l10n %}
+{% load cache %}
 
+{% cache 30 question question idea.id %}
```

Comment:
is there a `question` variable in the context here, or is that an accidental duplication of the cache name?